e2e/execution_test.go

```diff
@@ -89,6 +101,87 @@ func testExecution(t *testing.T) {
 		})
 	})
 
+	t.Run("double execution", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("??? execution", func(t *testing.T) {
```

What this test is doing - the description dosen't give a hint what's going on here.

Reply:
i removed this test as its redundant with "many executions in parallel" test